documentation/documentation/i18n/nl/docusaurus-plugin-content-pages/markdown-page.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Markdown page example
```

Comment:
maybe remove this file

Reply:
https://github.com/SELab-2/UGent-3/pull/250/commits/a121fc85325197eb439fe366384da23ec6b3bf53